module/VuFind/src/VuFind/Security/CspHeaderGenerator.php

```diff
@@ -124,4 +132,41 @@ protected function createHeaderObject()
             ? new ContentSecurityPolicyReportOnly()
             : new ContentSecurityPolicy();
     }
+
+    /**
+     * Create Report-To header based on given configuration
+     *
+     * @return >GenericHeader
```

Comment:
I think this is a typo -- `>` should be `?` -- unless I'm missing something. :-)

Reply:
This is incredibly advanced syntax I pulled in from PHP 14 but since you haven't learned it yet I changed it to a question mark. :-)